pkg/manager/member/tidb_member_manager.go

```diff
@@ -268,22 +271,41 @@ func (tmm *tidbMemberManager) syncTiDBServerCerts(tc *v1alpha1.TidbCluster) erro
 	suffix := "tidb-server"
 	ns := tc.GetNamespace()
 	tcName := tc.GetName()
-	svcName := fmt.Sprintf("%s-%s", tcName, suffix)
+	svcName := controller.TiDBMemberName(tcName)
```

Comment:
I think the Service name is `<tcName>-tidb`? If so, just change `suffix` to `tidb` would work, otherwise the `suffix` variable is no longer used.

Reply:
The `suffix` is used to construct the CSR Name, it must be distinguished from the tidb client certificate: https://github.com/pingcap/tidb-operator/blob/056339dea0c2ac19fd63a21bf6f113ef5bb0921f/pkg/manager/member/tidb_member_manager.go#L319